### ISSUE TITLE
Ordonnancement des Opérations et Tâches

### ISSUE THREAD

**Issue by waldo2188** [dbarzin/mercator#155]
Bonjour,

Avant tout, merci pour cette application qui m'est d'une grande aide.
N’étant pas expert du domaine, je ne sais pas si cette demande est pertinente.
Que pensez-vous de la possibilité d’ordonnancer les Opérations pour une activité et d’ordonnancer les Tâches pour une Opération ?

Je me doute que l’impact sur le code et la BDD est conséquent, mais je souhaitais vous soumettre l’idée.

En vous remerciant.

**Comment by dbarzin** [comment]
Qu'entends-tu pas ordonnancer ? Définir un ordre, donc ajouter un numéro de séquence en plus du nom ?

**Comment by waldo2188** [comment]
Oui, c'est tout a fait cela. Ajouter un ordre d'exécution des Tâches et des Opération. et donc pouvoir exprimer que dans l'Activité A1 l'Opération OA se fait avant l'opération OB et dans l'Activité A2, OB se fait avant l'opération OA.
Même système pour les Tâches.

**Comment by dbarzin** [comment]
Dans un premier temps, tu peux mettre le numéro de séquence dans le nom de la tâche ou de l'opération. Comme elles sont triées pas nom, cela devrait permettre de les ordonnancer.

**Comment by waldo2188** [comment]
C'est une solution que j'avais envisagé, mais je suis souvent limité par la restriction de 32 caractères maximum.
Le fonctionnement actuel n'est pas bloquant à l'usage que nous faisons du logiciel. Et lorsque cette fonctionnalité sortira (si bien sûr elle est jugé réalisable), je mettrais les processus à jour pour refléter au mieux la réalité.

**Comment by dbarzin** [comment]
L'ordonnancement des opérations et des tâches ne sera pas implémenté. Mercator fait l'inventaire de ces objets, mais ne décrits pas leurs interactions. Pour cela, il faut se diriger vers des outils qui font du Togaf ou de l'UML.

**Comment by lynch4fr** [comment]
Bonjour,
Nous avions discuté dans un autre poste d'un sujet similaire, la fonctionnalité semblait envisagée, elle pourra peut être vous correspondre. [https://github.com/dbarzin/mercator/discussions/970](url)
